CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+### Fixes
```

Comment:
I think we don't need a changelog for that, because the "broken" version hasn't been published yet, so no user-facing changes. So I'd just add _#skip-changelog_ to the PR description and remove it

Reply:
Sounds good 👍